README.md

```diff
@@ -63,41 +59,44 @@ df = spark.createDataFrame(data=embeddings, schema=schema)
     .mode("append")
     .save()
 )
-
-
 ```
 
 ### Scala
-
 ```scala
+import io.pinecone.spark.pinecone.PineconeOptions
 import org.apache.spark.SparkConf
-import org.apache.spark.sql.types._
+import org.apache.spark.sql.types.{ArrayType, FloatType, StringType, StructType}
```

Comment:
Getting away from global imports is good, but it seems like we aren't using these types anymore.

Reply:
Yes, that's an error! I replaced it with COMMON_SCHEMA so the import needed to be updated. Thanks!